BUILD

```diff
@@ -323,39 +323,36 @@ grpc_cc_library(
 )
 
 grpc_cc_library(
-    name = "grpc",
+    name = "grpc_init",
```

Comment:
The `srcs` is duplicate with the next target `grpc`. Is this intentional? How about have a dependency from `grpc` to `grpc_init` and remove `init.cc` from `grpc`

Reply:
Because init.cc is referenced by many rules, it's a bit messy to clean them up. So let's remove grpc_init and grpc_cronet which haven't been used simply.